User-Help/CodidactCommentFlagTexts.md

```diff
@@ -0,0 +1,31 @@
+## Flag reasons and texts
+
+**Why are you flagging this comment?**
+
+Please select a flag reason...
+
+**Code of Conduct violation**
+
+This comment is rude, disrespectful, or otherwise violates our [Code of Conduct](/policy/code-of-conduct).
+
+**Obsolete**
+
+This comment has been addressed via edits or other changes and is obsolete.
+
+**Other...**
+
+This comment needs moderator attention for a reason not covered by other options.
+
+**Placeholder text**: I'm flagging this comment because...
+
+## Response on flag action
+
+Thanks for flagging! A moderator will review this comment shortly.
```

Comment:
Maybe we shouldn't make time commitments ("shortly").  We try, but...

Reply:
I don't think it's a problem to include "shortly". It's not an explicit time commitment (e.g. "within the next week"), but it does give the message "this is on the list to get looked at".